src/pages/studies/StudyPageContainer.tsx

```diff
@@ -13,12 +13,27 @@ import { StudiesPageContainerData } from '../../store/graphql/studies/actions';
 import StudyIcon from 'icons/StudyIconSvg';
 
 import styles from './StudiesPageContainer.module.scss';
+import { IDictionary } from '@ferlab/ui/core/components/QueryBuilder/types';
 
 type StudyPageContainerProps = StudiesPageContainerData & PaginationType;
 
 const StudyPageContainer = ({ studiesResults, filters, pagination }: StudyPageContainerProps) => {
   const total = studiesResults?.data?.hits?.total || 0;
 
+  const labelByFieldKeyMap: Record<string, string> = {
```

Comment:
Not sure about the strategy here. There is already the Display Name that you can use from querying the arranger mapping. (extended mapping). See `studiesMappingResults ` in studies.tsx. Maybe use this to build the dictionary?

Reply:
Thanks for your comment, I was not aware of this mapping, I made changes to use it :)